.github/workflows/test.yml

```diff
@@ -42,4 +42,4 @@ jobs:
         if: startsWith(matrix.action, 'install')
 
       - name: Run chart-testing (${{ matrix.action }})
-        run: ct ${{ matrix.action }} --config config.yaml --debug
+        run: ct ${{ matrix.action }} --config config.yaml --debug --helm-extra-args '--timeout 10m'
```

Comment:
Fine for me, but just so you know, this can also be added in the [`config.yaml`](https://github.com/netbox-community/netbox-chart/blob/a844f113ea88285d5d00fb6cb169dbfee7261e85/config.yaml) file.
Eventually easier to manage.

```yaml
helm-extra-args: --timeout 10m
```

Reply:
@LeoColomb I can't for the life of me figure out where this is documented. Is it just anything that's a top-level arg has an equivalent entry in `config.yaml`?

Reply:
> All command-line flags can also be set via environment variables or config file.
>
> _https://github.com/helm/chart-testing/blob/9075fa18dd714a9dea70c5f9df8395f4891af1dc/README.md#configuration_

😊

Reply:
Ah, so it's a `chart-testing` feature, not a feature of Helm. That would explain my inability to find it in Helm's upstream documentation. 😅